docs/interfaces/CC++.rst

```diff
@@ -1,6 +1,6 @@
-.. _c_cpp_interface:
+.. _c_interface:
 
-C/C++
+C
```

Comment:
How about we also change the name of the file to `C.rst`?

Reply:
Done here - will send separate PR for master.